components/jaggery-core/org.jaggeryjs.jaggery.core/src/main/resources/META-INF/scripts/http.js

```diff
@@ -14,7 +14,7 @@ var get, post, put, del, head, options, trace, connect;
         if (type === "xml") {
 			return new XML(data.replace(/<\?xml.*?\?>/, "").replace(/<!--[\s\S]*?-->/g, ""));
 		} else if (type === "json") {
-			return parse(data);
+			return JSON.parse(data);
```

Comment:
It is not the usual practice to change parts of the file, that does not need change when doing bug fixes. Specially in this case, since the issue in question is related to formatting. IMO, formatting fixes, if any should come as separate PRs.